server/model/monitor.js

```diff
@@ -547,12 +559,17 @@ class Monitor extends BeanModel {
             log.debug("monitor", `[${this.name}] Store`);
             await R.store(bean);
 
+            if (this.isEnabledSlowResponseNotification()) {
+                log.debug("monitor", `[${this.name}] Check response is slow`);
+                await this.checkSlowResponseNotification(this);
+            }
+
             log.debug("monitor", `[${this.name}] prometheus.update`);
             prometheus.update(bean, tlsInfo);
 
             previousBeat = bean;
 
-            if (! this.isStop) {
+            if (!this.isStop) {
```

Comment:
Looking at the rest of the codebase, `! `  seems to be the standard.
```suggestion
            if (! this.isStop) {
```